website/docs/automation/controlplanes/ack/provision-resources.md

```diff
@@ -1,70 +1,93 @@
 ---
-title: "Provision ACK Resources"
+title: "Provisionining ACK Resources"
 sidebar_position: 5
 ---
 
-By default the catalog component in the sample application uses a MySQL database running as a pod in the EKS cluster. In this lab, we'll provision an Amazon RDS database for our application using Kubernetes custom resources to specify the desired configuration required by the workload.
+By default the **Carts** component in the sample application uses a DynamoDB local instance running as a pod in the EKS cluster called ```carts-dynamodb```. In this section of the lab, we'll provision an Amazon DynamoDB cloud based table for our application using Kubernetes custom resources and point the **Carts** deployment to use the newly provisioned DynamoDB table instead of the local copy.
 
-![ACK reconciler concept](./assets/ack-desired-current.jpg)
+![ACK reconciler concept](./assets/ack-desired-current-ddb.png)
 
-The first thing we need to do is create a Kubernetes secret that'll be used to provide the master password for the RDS database. We'll configure ACK to read this secret as a source for the password:
+The AWS Java SDK in the **Carts** component is able to use IAM Roles to interact with AWS services which means that we do not need to pass credentials, thus reducing the attack surfacec. In the EKS context, IRSA allows us to define per pod IAM Roles for applications to consume. To leverage IRSA, we first need to:
```

Comment:
It looks like originally you had the Terraform create the IRSA role etc, and I would prefer that. I would suggest we only make a basic reference to the fact that IRSA is used here and direct the user to the IRSA lab if they want to learn more.

We definitely don't want a screenshot of the terminal (harder to maintain) and we tend to get the user to run AWS CLI commands to retrieve things like the trust policy if we want to show them it. Regardless I think most of this can be removed.

Reply:
@niallthomson I insisted we show the IRSA creation here. This is different than installing the controller because that would be part of the platform team responsibility. The idea here is not to obscure that even though the dynamo controller can be used to create the dynamo table, the developer still needs to set the permissions. I'd even prefer to create the trust policy with the aws cli but showing it would be ok too.

Reply:
Cool thank you very much for the feedback! I've implemented all the changes suggested in the comments.